.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Workspace-specific settings override global settings, so if you've set up your own formatter, it will be overridden. I'm ok with that because CI _will_ fail if you haven't formatted as prettier would.